lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1681,7 +1693,7 @@ if (!sourceList)
         }
 
         ParseNodePtr parseTree;
-        if ((loadScriptFlag & LoadScriptFlag_Utf8Source) == LoadScriptFlag_Utf8Source)
```

Comment:
> What happens to this flag now? Should we use it at least for assert purposes?

I consolidated it into "utf8FromExternal" (which means it is utf8 and fromExternal). As discussed off line, this may be temporary, depending on how we are going to treat CESU8. (Can we drop CESU8?)

Yes I can add an assertion.